src/FootTable.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { measureScrollbar } from './utils';
+import BaseTable from './BaseTable';
+
+export default function FootTable(props, { table }) {
+  const { prefixCls, scroll, renderFooterInTable } = table.props;
+  const { columns, fixed, tableClassName, handleBodyScrollLeft, expander } = props;
+  const { saveRef } = table;
+  let { useFixedHeader } = table.props;
+  const footStyle = {};
+
+  if (scroll.y) {
+    useFixedHeader = true;
+    // Add negative margin bottom for scroll bar overflow bug
+    const scrollbarWidth = measureScrollbar();
+    if (scrollbarWidth > 0 && !fixed) {
+      footStyle.marginBottom = `-${scrollbarWidth}px`;
+      footStyle.paddingBottom = '0px';
+    }
+  }
+
+  if (!useFixedHeader || !renderFooterInTable) {
+    return null;
+  }
+
+  return (
+    <div
+      key="footTable"
+      ref={fixed ? null : saveRef('footTable')}
+      className={`${prefixCls}-footer`}
+      style={footStyle}
+      onScroll={handleBodyScrollLeft}
+    >
+      <BaseTable
```

Comment:
Why render `tfoot` to a separator `BaseTable`, can we just render it in the `BaseTable`?

Reply:
Reason is to keep its behavior consistent to Header's. When you turn `{ scroll: y }` on, Header is fixed, most likely user want the Footer to be fixed too.